pages/facts.html

```diff
@@ -1,12 +1,65 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link rel="stylesheet" href="../styles/style.css">
+  <link rel="stylesheet" href="../styles/facts.css">
+  <link rel="icon" type="image/x-icon" href="../images/logo.gif">
+  <title>Fun Facts About Coffee</title>
 </head>
+
 <body>
-  
+  <header>
+    <nav>
+      <a href="../index.html">Home</a>
+      <a href="facts.html" class="active">Fun Facts</a>
+      <a href="gallery.html">Gallery</a>
+      <a href="about_us.html">About Us</a>
+    </nav>
+  </header>
+  <main>
+    <section>
+      <div class="logo">
+        <img src="../images/logo.gif" alt="logo">
+      </div>
+      <h1>Welcome to Coffee Facts!</h1>
+      <h2>Fun Facts About Coffee</h2>
+      <div class="grid-container">
+        <div class="grid-item">
+          <p><strong>Most Popular Drink</strong> Coffee is one of the most popular drinks worldwide, with over 2
+            billion cups consumed each day.</p>
+        </div>
```

Comment:
Since the fun facts are standalone content, we could use `article` tags to make your html more semantic.